### PR TITLE
Fix flake8 errors

### PR DESCRIPTION
**Pull request by natifridman**

### CODE REVIEW

File: src/vse_sync_pp/parsers/phc2sys.py [hunk 1]
```diff
@@ -7,31 +7,35 @@
...
     y_name = 'terror'
     parsed = namedtuple('Parsed', elems)
+
     @staticmethod
     def build_regexp():
         """Return a regular expression string for parsing phc2sys log file lines"""
         return r'\s'.join((
-            r'^phc2sys' +
-            r'\[([1-9][0-9]*\.[0-9]{3})\]:'+  # timestamp
+            r'^phc2sys' + # noqa: W504
```
Comment (nocturnalastro) [inline src/vse_sync_pp/parsers/phc2sys.py:22]: Instead of adding a noqa I'd move `+`

File: src/vse_sync_pp/parsers/parser.py [hunk 2]
```diff
@@ -64,6 +68,7 @@ def relative_timestamp(parsed, tzero):
         parsed = parsed._replace(timestamp=timestamp - tzero)
     return tzero, parsed
 
+
 class Parser():
     """A base class providing common parser functionality"""
     def make_parsed(self, elems): # pylint: disable=no-self-use
```
Comment (jnunyez) [inline src/vse_sync_pp/parsers/parser.py:74]: If we are going with flake8 I would take out all lines we previously added  based on adding `# pylint` ?

File: src/vse_sync_pp/parsers/phc2sys.py (hunk 1 above)
Comment (natifridman) [reply src/vse_sync_pp/parsers/phc2sys.py]: Fixed, added the opposite `W503` to the ignored list

File: tests/vse_sync_pp/parsers/test_gnss.py [hunk 3]
```diff
@@ -11,30 +11,23 @@
...
-        ),
-        (
-            '2023-06-16T17:01:11.131282269+00:00,3,398',
-            (Decimal('1686934871.131282269'), 3, 398),
-        ),
-        (   '681011.839,5,-3,-2.72',
-            (Decimal('681011.839'), 5, -3),
-        ),
+        ('681011.839,5,-3',
+         (Decimal('681011.839'), 5, -3),),
```
[the remark this answers is not in the document]
Comment (natifridman) [reply tests/vse_sync_pp/parsers/test_gnss.py]: Fixed

File: src/vse_sync_pp/parsers/parser.py (hunk 2 above)
Comment (natifridman) [reply src/vse_sync_pp/parsers/parser.py]: Removed

**Review by jnunyez — APPROVED** [review]

**Pull request merged**